src/test/java/teammates/test/util/Url.java

```diff
@@ -0,0 +1,16 @@
+package teammates.test.util;
+
+import teammates.test.driver.TestProperties;
+
+public class Url extends teammates.common.util.Url {
```

Comment:
I prefer to have a more specific name for this class. Sub classes usually have different names from the parent class.